config.js

```diff
@@ -3,6 +3,6 @@
  * https://beta.developer.spotify.com/dashboard
  */
 
-const client_id = 'YOUR_CLIENT_ID';
-const redirect_uri = 'YOUR_CALLBACK_URL';
-const client_secret = 'YOUR_CLIENT_SECRET';
\ No newline at end of file
+const client_id = '89d30ad7388e4d52a61ada58a19c8928';
+const redirect_uri = 'http://localhost:3000/callback';
+const client_secret = 'd7dda67f232541ae8433eb62dbb9df22';
```

Comment:
you left some config stuff in, including your client ID and secret

Reply:
Yeah my bad, I've removed and deactivated the secrets.